predicators/refinement_estimators/base_refinement_estimator.py

```diff
@@ -4,12 +4,17 @@
 from pathlib import Path
 from typing import List, Set
 
-from predicators.structs import GroundAtom, State, _GroundNSRT
+from predicators.envs import get_or_create_env
+from predicators.settings import CFG
+from predicators.structs import GroundAtom, Task, _GroundNSRT
 
 
 class BaseRefinementEstimator(abc.ABC):
     """Base refinement cost estimator."""
 
+    def __init__(self) -> None:
+        self._env = get_or_create_env(CFG.env)
```

Comment:
this is only needed for rendering right? if so, can we make that clear by just storing the render function here instead of the whole environment?

Reply:
It's also used to get env.types in the oracle estimator